CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 Documentation for hipRAND is available at
 [https://rocm.docs.amd.com/projects/hipRAND/en/latest/](https://rocm.docs.amd.com/projects/hipRAND/en/latest/).
 
+## hipRAND-2.11.1 for ROCm 6.2.4
```

Comment:
Should this be 6.3.0?

Reply:
There are no updates for ROCm 6.3.0, hence no entry for it.  This is merely to add the 6.2.4 entry into the 6.3 changelog for completeness.

Reply:
Thanks. Understood.